ui/js/models/perf/alerts.js

```diff
@@ -93,21 +98,23 @@ treeherder.factory('PhAlerts', [
                 if (copySummary) {
                     resultStr += "\n";
                 }
-                resultStr += "Summary of tests that regressed:\n\n" +
+                resultStr += "== Regressions ==\n\n" +
                              _.map(regressed, function(alert) {
-                                 return "  " + alert.title + ": " + alert.prev_value + " -> " +
-                                 alert.new_value + " (" + alert.amount_pct + "% worse)";
+                                 return _.padLeft(alert.amount_pct.toFixed(0), 3) + "%  " +
+                                  _.padRight(alert.title + ": ", _.max(regressed, function(alert){ return alert.title.length; }).title.length +5) +
```

Comment:
It might actually be worth factoring this out into an inline function called formatAlertList (or something) and reusing it both here and below for the improvements. The logic has gotten complex enough that it's probably justified. :)